projectFilesManager.js

```diff
@@ -37,6 +37,21 @@ function forceUpdateProjectFiles(projectDir, appDir) {
     addProjectFiles(projectDir, appDir);
 }
 
+function compareProjectFiles(projectDir) {
+    const projectTemplates = getProjectTemplates(projectDir);
+    Object.keys(projectTemplates).forEach(function(newTemplatePath){
+        const currentTemplatePath = projectTemplates[newTemplatePath];
+        if (fs.existsSync(currentTemplatePath)) {
+            const currentTemplate = fs.readFileSync(currentTemplatePath).toString();
+            const newTemplate = fs.readFileSync(newTemplatePath).toString();
+            if (newTemplate !== currentTemplate) {
+                const message = `The current project contains a ${path.basename(currentTemplatePath)} file located at ${currentTemplatePath} that differs from the one in the new version of the nativescript-dev-webpack plugin located at ${newTemplatePath}. Some of the plugin features may not work as expected until you manually update the ${path.basename(currentTemplatePath)} file.`;
+                console.info(`\x1B[33;1m${message}\x1B[0m` );
```

Comment:
Developers can also use this script to overwrite the existing configuration files:

```
./node_modules/.bin/update-ns-webpack --configs
```